e2e/testsuite/testsuite.go

```diff
@@ -76,8 +76,8 @@ type GRPCClients struct {
 	InterTxQueryClient intertxtypes.QueryClient
 
 	// SDK query clients
-	GovQueryClient   govtypesv1beta1.QueryClient
-	GovQueryClientV1 govtypesv1.QueryClient
+	GovQueryClient    govtypesv1beta1.QueryClient
+	GovQueryClientV1  govtypesv1.QueryClient
```

Comment:
oh weird this kinda seems like we're not auto formatting the test code if linting was passing without the correct indentation.